Debian Stable build bot is failing with the following error:

https://build.webkit.org/#/builders/46/builds/26034

```
WTF/Headers/wtf/RefPtr.h: In instantiation of ‘static void WTF::DefaultRefDerefTraits< <template-parameter-1-1> >::derefIfNotNull(T*) [with T = WebCore::VideoFrame]’:
WTF/Headers/wtf/RefPtr.h:74:61:   required from ‘WTF::RefPtr<T, <template-parameter-1-2>, <template-parameter-1-3> >::~RefPtr() [with T = WebCore::VideoFrame; _PtrTraits = WTF::RawPtrTraits<WebCore::VideoFrame>; _RefDerefTraits = WTF::DefaultRefDerefTraits<WebCore::VideoFrame>]’
../../Source/WebCore/html/canvas/WebGLRenderingContextBase.cpp:1089:16:   required from here
WTF/Headers/wtf/RefPtr.h:42:18: error: invalid use of incomplete type ‘class WebCore::VideoFrame’
   42 |             ptr->deref();
      |             ~~~~~^~~~~
In file included from ../../Source/WebCore/html/canvas/WebGLRenderingContextBase.h:33,
                 from ../../Source/WebCore/html/canvas/WebGLExtension.h:30,
                 from ../../Source/WebCore/html/canvas/WebGLMultiDrawInstancedBaseVertexBaseInstance.h:28,
                 from ../../Source/WebCore/html/canvas/WebGLMultiDrawInstancedBaseVertexBaseInstance.cpp:29,
                 from WebCore/DerivedSources/unified-sources/UnifiedSource-950a39b6-30.cpp:1:
```

--- a/Source/WebCore/html/canvas/WebGLRenderingContextBase.cpp
+++ b/Source/WebCore/html/canvas/WebGLRenderingContextBase.cpp
@@ -144,6 +144,10 @@
 #include <wtf/text/CString.h>
 #include <wtf/text/StringBuilder.h>
 
+#if ENABLE(MEDIA_STREAM)
+#include "VideoFrame.h"
+#endif
+
 #if ENABLE(OFFSCREEN_CANVAS)
 #include "OffscreenCanvas.h"
 #endif

--- a/Source/WebCore/testing/Internals.cpp
+++ b/Source/WebCore/testing/Internals.cpp
@@ -299,6 +299,7 @@
 #if ENABLE(MEDIA_STREAM)
 #include "MediaStream.h"
 #include "MockRealtimeMediaSourceCenter.h"
+#include "VideoFrame.h"
 #endif
 
 #if ENABLE(MEDIA_RECORDER)

